### PR TITLE
Blog post for the September 2016 roundup

### PR DESCRIPTION
**Pull request by Haacked**
This is a blog post listing the apps added to the site in September :sparkles:

### CODE REVIEW

File: _posts/2016-10-05-september-2016-roundup.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+| <img src='/images/apps/qickcalcicon.png' width='50'> | [quickcalc](https://github.com/Cwoodall6/quickcalc) | Menubar Calculator |
+| <img src='/images/apps/forestpin.png' width='50'> | [Forestpin Analytics](http://forestpin.com/analytics) | Financial data analytics tool for businesses |
+| <img src='/images/apps/ling.png' width='50'> | [Ling](https://github.com/talhasch/ling) | REST Client |
+| <img src='/images/apps/shortexts.png' width='50'> | [Shortexts](http://shortexts.com/) | Shortcuts for texts you copy frequently, folders and emojis |
+| <img src='/images/apps/front-end-box.png' width='50'> | [Front-End Box](http://frontendbox.io) | Set of front-end-code generators |
+
+
+
+
+### New Meetups
```
Comment (kevinsawicki) [inline _posts/2016-10-05-september-2016-roundup.md]: Maybe :fire: this section since it is empty

Comment (Haacked) [inline _posts/2016-10-05-september-2016-roundup.md]: Oh yeah! That's what I get for not checking the template over.

**Comment by Haacked** [comment]
I added a couple of Electron talks from GitHub Universe and plugged @zeke's upcoming dotJS talk.

**Comment by zeke** [comment]
Nice. When should this go out?

**Comment by Haacked** [comment]
> Nice. When should this go out?

It could go out now if we want. Do we typically schedule these things for certain days/times?

**Comment by kevinsawicki** [comment]
> It could go out now if we want. Do we typically schedule these things for certain days/times?

Usually 9am-11am PST, little late in the day for east coast right now, tomorrow morning sounds good to me 👍

**Comment by Haacked** [comment]
> Usually 9am-11am PST, little late in the day for east coast right now, tomorrow morning sounds good to me

Ok, i set myself a reminder to publish this tomorrow morning.

**Pull request merged**